lib/chef/resource/chocolatey_package.rb

```diff
@@ -0,0 +1,34 @@
+#
```

Comment:
Not sure if this file is necessary at all. I believe if we derrive from windows_package, this can go.